integration/auth/auth_test.go

```diff
@@ -78,97 +76,87 @@ func TestCustomToken(t *testing.T) {
 	if vt.UID != "user1" {
 		t.Errorf("UID = %q; want UID = %q", vt.UID, "user1")
 	}
-	if err = client.DeleteUser(context.Background(), "user1"); err != nil {
-		t.Error(err)
+}
+
+func TestCustomTokenWithClaims(t *testing.T) {
+	ct, err := client.CustomTokenWithClaims(context.Background(), "user2", map[string]interface{}{
```

Comment:
Do you want to use random UIDs on all these other tests?

Reply:
Done